build-system/common/git.js

```diff
@@ -169,12 +169,30 @@ function gitCommitterEmail() {
 }
 
 /**
- * Returns the timestamp of the latest commit on the local branch.
+ * Returns list of commit SHAs and their cherry-pick status from master.
```

Comment:
Expand this JSDoc description to briefly mention that `- <sha>` indicates a cherry-picked commit while `+ <sha>` indicates a new commit on the branch.

Reply:
Done